### DISCUSSION TITLE
Enable access to the global npm only for specific user

### DISCUSSION THREAD

**Discussion by mhalp** [verdaccio/verdaccio#2030] [Q&A] [answered]
Hi,
I want to define that only specific users will be able to access the global npm through verddacio,
and all the other users will be able to use only the verdaccio local packages
How can I define it?

**Comment by juanpicado** [comment, marked as answer]
do these guides help? or maybe you can describe your situation a bit more

https://verdaccio.org/docs/en/best
https://verdaccio.org/docs/en/protect-your-dependencies
